backend/scripts/backfill.js

```diff
@@ -181,7 +181,7 @@ async function handleEvents(listingIdFull) {
     order: [['block_number', 'ASC']]
   })
   for (const event of events) {
-    await insertOrderFromEvent({
+    await processDShopEvent({
       offerId: `${listingIdFull}-${event.offerId}`,
```

Comment:
I guess we don't need this any more since it may not be an offer event?

Reply:
Good point. Removed the offerId argument from this method.